packages/tasit-action/src/contract/Action.test.js

```diff
@@ -0,0 +1,12 @@
+import Action from "./Action";
+import Subscription from "./Subscription";
+
+describe("subscribing to an event before sending tx", () => {
+
+    it("gives error if no subscribed events", () => {
+        if(this.subscribedEventNames() === []){
+            assert.ifError(this.on("confirmation", () => {}));
```

Comment:
Hi @zbash75, we are using [`expect`](https://www.chaijs.com/api/bdd/) to assertion on our test suites.